packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -3,12 +3,17 @@
 import { BodyText, Button, Icon, RadioButton, TitleText } from "@/ui";
 import { useState } from "react";
 import { ConfirmStackModal } from "./ConfirmStackModal";
-import { TokenPicker } from "@/components/token-picker/TokenPicker";
+import { TokenPicker, DatePicker } from "@/components";
 
 export const Stackbox = () => {
   const [isConfirmStackOpen, setConfirmStackIsOpen] = useState(false);
   const [isTokenPickerOpen, setTokenPickerIsOpen] = useState(false);
 
+  const [frequency, setFrequency] = useState<string>("0");
+
+  const [startDateTime, setStartDateTime] = useState<Date>(new Date());
+  const [endDateTime, setEndDateTime] = useState<Date>(new Date());
```

Comment:
Can we set the default end time date 1 month from now?

I think it would be nice to already be in the future, one month seems nice.

something like this should work (from a quick search)
```suggestion
  const [endDateTime, setEndDateTime] = useState<Date(new Date().setMonth(new Date().getMonth() + 1);
```

Reply:
This didn't work but applied another solution. Thanls!